gluster/block/README.md

```diff
@@ -1,31 +1,37 @@
 # glusterblock Volume Provisioner for Kubernetes 1.5+
 
-
 [![Docker Repository on Quay](https://quay.io/repository/external_storage/glusterblock-provisioner/status "Docker Repository on Quay")](https://quay.io/repository/external_storage/glusterblock-provisioner)
+
+Latest stable container image can be pulled from:
+
 ```
 quay.io/external_storage/glusterblock-provisioner:latest
 ```
 
-# Test instruction
+# Build Gluster Block Provisioner
+
+If you have `go` envrionment configured, you can build your own binary or container image as shown below:
 
-* Build glusterblock-provisioner and container image
+#### Clone and build binary and docker image
```

Comment:
Why a level-4 heading after a level-2 heading? In PR #123, I use level 2, and this looks very consistent to me.